src/languages/cs_CZ/commands/ping.json

```diff
@@ -0,0 +1,3 @@
+{
+	"DESCRIPTION": "Zjistí jestli bot odpovídá."
+}
```

Comment:
newline?

Reply:
Are you sure there should be new line in json file? Visual Studio Codium automatically removes it on save. Is this correct?

Reply:
Well I don't like no new lines at the end. Also my formatter adds newlines on json too.

Reply:
Ok, let me see how to change it